src/util/json_stream.h

```diff
@@ -169,6 +169,20 @@ class json_stream_objectt : public json_streamt
       return it->second;
   }
 
+  /// Push back a JSON element into the current array stream.
```

Comment:
object stream

Reply:
Add a comment that the element pushed this way cannot be accessed through [] anymore because it has been output already.